pkg/data/data.go

```diff
@@ -136,12 +145,83 @@ func (ff *FieldFilter) Accept(ks *KeySummary) (bool, error) {
 	}
 }
 
+// HashByRevision returns the checksum and revision that was checksumed of a etcd keyspace.
```

Comment:
Let's document more specifically what is included in the hash: only the latest version of each key, only user key-value. Because this is quite different than the one in etcd server.

Reply:
And maybe document the fact that the resulting hash is more consistent, in the sense that it does not depend on compact revision in db file.

Reply:
Added